### PR TITLE
Feature/list view border color

### LINKED ISSUE #4298: Add border color property for outer border to the list view styles
### Is your proposal related to a problem?

Currently, There's no option to set a border color to the outer border of the list view component.

### Describe the solution you'd like

Add an Outer border-color property to the list view component.

Screenshot 2022-10-04 at 4 42 28 PM

### PR DESCRIPTION
**Pull request by FroostyCodes**
Fixes #4298

### CODE REVIEW

**Review by arpitnath — CHANGES_REQUESTED** [review]
Please add a default value in the widget definitions in the **widgetConfig.js**

**Comment by FroostyCodes** [comment]
@arpitnath I have made the changes you requested, do let me know if any further changes are to be done.

**Review by arpitnath — APPROVED** [review]

File: frontend/src/Editor/Components/Listview.jsx [hunk 1]
```diff
@@ -22,8 +22,12 @@ export const Listview = function Listview({
   const { visibility, disabledState, borderRadius } = { ...fallbackStyles, ...styles };
   const backgroundColor =
     ['#fff', '#ffffffff'].includes(styles.backgroundColor) && darkMode ? '#232E3C' : styles.backgroundColor;
+  const borderColor = styles.borderColor;
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Components/Listview.jsx:25]:
```suggestion
  const borderColor = styles.borderColor ?? 'transparent';
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@FroostyCodes Can you add an undefined check with transparent color? This way it won't affect the existing apps.

**Comment by FroostyCodes** [comment]
@kavinvenkatachalam done

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**